pkg/operator/profilecalculator.go

```diff
@@ -151,12 +151,13 @@ func (pc *ProfileCalculator) nodeChangeHandler(nodeName string) (bool, error) {
 }
 
 type ComputedProfile struct {
-	TunedProfileName string
-	AllProfiles      []tunedv1.TunedProfile
-	Deferred         util.DeferMode
-	MCLabels         map[string]string
-	NodePoolName     string
-	Operand          tunedv1.OperandConfig
+	TunedProfileName     string
+	AllProfiles          []tunedv1.TunedProfile
+	Deferred             util.DeferMode
+	MCLabels             map[string]string
+	NodePoolName         string
+	Operand              tunedv1.OperandConfig
+	TuneDProfileConflict []string
```

Comment:
Is storing this really necessary? The AllProfiles list already has this information and it is enough to just report the duplicate names in the status I'd say?

Reply:
You'd have to go through the AllProfiles list *again* when calculating the status and check their content.  Same content is not a problem and I don't think we should make co/node-tuning degraded.  The "cache" is an optimization.  Data vs. CPU cycles.  I'm for saving CPU cycles.